views/tasks.py

```diff
@@ -65,3 +66,9 @@ def notify_subscribers():
     notifier = logic.notifier.notifier_for_event(request.json['event'])(**request.json['options'])
     notifier.notify()
     return Response(status=200)
+
+
+@app.route('/tasks/lovelinks/cleanup', methods=['GET'])
```

Comment:
agree with this. alinar, I thought we talked about GET/POST for the endpoint. was there a specific reason you used a GET here?